vocabularies/Common.xml

```diff
@@ -124,6 +124,16 @@
         <Annotation Term="Core.RequiresType" String="Edm.DateTimeOffset" />
       </Term>
 
+      <Term Name="Locale" Type="Edm.String" Nullable="true" AppliesTo="Property Parameter ReturnType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The value of the annotated property or parameter shall be interpreted in the given locale" />
```

Comment:
```suggestion
        <Annotation Term="Core.Description" String="The value of the annotated property, parameter, or return type must be interpreted in the given locale" />
```

Reply:
I cannot judge the precise difference between "shall" and "must", but I feel uncomfortable saying "You must ..." in connection with an annotation, because clients are always allowed to not understand a given annotation.

Reply:
"shall" is an outdated synonym for "must". Use "shall" if you want to sound old-fashioned, or like a biblical prophet 😄